model/Core/Classes/ElementCollection.md

```diff
@@ -19,10 +19,10 @@ An SpdxCollection is a collection of Elements, not necessarily with unifying con
 ## Properties
 
 - element
-  - type: Element
+  - type: Element and NOT (X-Collection)
```

Comment:
Note: the spec parser does not currently support this expression

Reply:
That does not surprise me.
It will need to though.
This sort of issue I why I strongly prefer having the formal ontology specification (RDFS/OWL/SHACL) be THE ground truth specification rather than a prose form that must be converted. 
Specifying this sort of range is simple, native and inherent in the RDFS/OWL/SHACL.